services/registration/controller/controller.go

```diff
@@ -291,19 +291,19 @@ func CreateCurrentMentorRegistration(w http.ResponseWriter, r *http.Request) {
 	err = service.CreateMentorRegistration(id, mentor_registration)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.InternalError(err.Error(), "Could not create mentor registration.\n"+err.Error()))
 	}
 
 	err = service.AddMentorRole(id)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.InternalError(err.Error(), "Could not add mentor role.\n"+err.Error()))
```

Comment:
Don't need a newline here and don't need to append `err.Error()` since it will be included in the `RawError` field.